shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -63,6 +63,7 @@ struct FlutterPlatformViewLayer {
   fml::scoped_nsobject<UIView> overlay_view;
   std::unique_ptr<IOSSurface> ios_surface;
   std::unique_ptr<Surface> surface;
+  void UpdateSurface();
```

Comment:
Forgot to implement?

Reply:
Removed - I thought I'd need this and didn't.